metastore/iceberg-metastore/src/test/java/org/apache/drill/metastore/iceberg/components/tables/TestIcebergTablesMetastore.java

```diff
@@ -297,6 +302,7 @@ public void testOverwrite() {
       .workspace(tableInfo.workspace())
       .tableName(tableInfo.name())
       .metadataKey("dir0")
+      .metadataType(MetadataType.TABLE.name())
```

Comment:
Some of these use the `MetadataType.TABLE`, others `MetadataType.TABLE.name()`. Do the tests make sure both work, or should they be uniform?

Reply:
In `TableMetadataUnit` most of the data is stored as string to be suitable for storage (for example, in the file), since this is unit test I had to create `TableMetadataUnit` manually and thus indicate metadata type string value, in the code we usually do not do this.